build-system/tasks/visual-diff.js

```diff
@@ -561,15 +575,27 @@ function setDebuggingLevel() {
   process.env.WEBSERVER_QUIET = '--quiet';
 
   if (argv.debug) {
-    // eslint-disable-next-line google-camelcase/google-camelcase
-    argv.chrome_debug = true;
-    process.env.WEBSERVER_QUIET = '';
+    argv.chrome_debug = true; // eslint-disable-line google-camelcase/google-camelcase
+    argv.gulp_debug = true; // eslint-disable-line google-camelcase/google-camelcase
+    argv.webserver_debug = true; // eslint-disable-line google-camelcase/google-camelcase
   }
   if (argv.webserver_debug) {
     process.env.WEBSERVER_QUIET = '';
   }
 }
 
+/**
+ * Return sub-process options based on whether its output should be printed.
+ *
+ * @param {boolean} shouldPrint true if the sub-process should print its output
+ * @return {<Object>} sub-process options
+ */
+function getStdioOptions(shouldPrint) {
```

Comment:
I don't think this function is required. See other comments.

Reply:
Done